internal/provider/svm/svm_resource.go

```diff
@@ -120,6 +122,12 @@ func (r *SvmResource) Schema(ctx context.Context, req resource.SchemaRequest, re
 				MarkdownDescription: "Maximum number of volumes that can be created on the svm. Expects an integer or unlimited",
 				Optional:            true,
 			},
+			"storage_limit": schema.Int64Attribute{
+				MarkdownDescription: "Maximum storage permitted on svm, in bytes",
+				Optional:            true,
+				Computed:            true,
+				Default:             int64default.StaticInt64(0),
```

Comment:
It is an optional parameter based on the API doc https://docs.netapp.com/us-en/ontap-restapi-9131/ontap/post-svm-svms.html#recommended-optional-properties
The default value is not needed.

Reply:
Without a default value, the attribute remains "unknown" after the apply - resulting in the following error:

```
╷
│ Error: Provider returned invalid result object after apply
│ 
│ After the apply operation, the provider still indicated an unknown value for netapp-ontap_svm.example.storage_limit. All values must be known after apply, so this is
│ always a bug in the provider and should be reported in the provider's own repository. Terraform will still save the other known object values in the state.
╵
```

I could manually set the value to 0 in `internal/provider/svm/svm_resource.go > Create()`, but isn't a default value in the schema definition easier to understand?

Reply:
You may use 				
```
        PlanModifiers: []planmodifier.Int64{
              IntUseStateForUnknown(),
       },
```
ref. https://github.com/NetApp/terraform-provider-netapp-ontap/blob/integration/main/internal/provider/storage/storage_aggregate_resource.go#L159 and see how to handle this parameter in creation/read/update. Since it's optional in the API call, the default value does not have to be set.

Reply:
Understood. I have removed the default value, and `omitempty` removes the parameter from the request body if unset.

I've adapted the same logic as in [StorageVolumeResource.PercentSnapshotSpace](https://github.com/NetApp/terraform-provider-netapp-ontap/blob/integration/main/internal/provider/storage/storage_volume_resource.go#L647).
Please let me know if this is acceptable.